.tours/get-started-with-dvc.tour

```diff
@@ -0,0 +1,77 @@
+{
```

Comment:
The main goal here was to make a low-cost prototype not something that is perfect.